### ISSUE TITLE
EPCO hardcoded to 0.9 in pl_waterup.f90

### ISSUE THREAD

**Issue by jenskiesel** [swat-model/swatplus#28]
I noticed that EPCO is not sensitive. Is there a reason why EPCO is set to 0.9 in this line:
https://github.com/swat-model/swatplus/blob/344efc6c0251c10df62fef18d3bb878869a3cf10/src/pl_waterup.f90#L134
Can I remove this line and commit?

**Comment by tugraskan** [comment]
Yes, this can be removed. I discussed it with Dr. Arnold, and it was hardcoded for national run testing purposes instead of using pl_prms calibration values because low values in it were causing errors. I will also remove this in the next release.

**Comment by jenskiesel** [comment]
Thanks for the information, @tugraskan, and for taking care of this!
